### ISSUE TITLE
router is being routed to london

### ISSUE THREAD

**Issue by Batman2814** [AdguardTeam/AdGuardDNS#627]
### Platform

Router

### Protocol

Regular

### Do you use AdGuard app?

No I don't

### Your configuration

I have AdGuard Private DNS setup on my Eero router where it's showing the protocol DNS server is going to "dns2-dp-lon-1". Not sure if this is London but if you can confirm for me where the location is where I'm also located in Canada.

This is related to issue #626 
https://github.com/AdguardTeam/AdGuardDNS/issues/626#issuecomment-1594855291

### Issue Details

N/A

### Actual Behavior

N/A

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

### Additional Information

I did restart my Eero router as you said but it is still being routed to server "dns2-dp-lon-4" from where I live in Canada. 
Can you keep this open until the issue is fixed?

**Comment by Batman2814** [comment]
Ignore the eero part name. Just means the Adguard Private DNS is set up on my router, and when I went to the Adguard services check page, it was routed to London from Canada.

**Comment by D13410N3** [comment]
Can you show traceroute results for 94.140.14.14?

**Comment by Batman2814** [comment]
Can you tell me how to do that?

**Comment by D13410N3** [comment]
On Windows: open command line (cmd) and execute command `tracert 94.140.14.14`
On Linux / MacOS: open terminal and execute command `traceroute 94.140.14.14` (if you don't have such command, install `traceroute` with your preferred package manager (i.e. `apt` on Debian / Ubuntu or `brew` on MacOS)

**Comment by Batman2814** [comment]
Don't you mean put in 94.140.14.49 for private dns?

**Comment by Batman2814** [comment]
![20230619_06325852_Edited](https://github.com/AdguardTeam/AdGuardDNS/assets/96619913/d0552e2b-a0e6-4832-92ba-4d1fc1997e55)

**Comment by D13410N3** [comment]
I've made some fixes, can you please check now?
Content of the page https://dns.adguard.com/info.txt and traceroute results again

**Comment by Batman2814** [comment]
Opened the linked on the device that connected to the router which shows its now connected to dns2-dp-chi-2

When I go to Adguard services check page it shows dns2-dp-la-1 

Is it connected to Chicago or LA?

**Comment by D13410N3** [comment]
What about now?

**Comment by Batman2814** [comment]
When clicking on the link says dns2-dp-chi-1 then on adguard services check page dns2-dp-la-1

**Comment by Batman2814** [comment]
when clicking on the link it still shows "dns2-dp-chi-2 and on AdGuard Services Check Page it's showing "dns2-dp-la-1"

**Comment by D13410N3** [comment]
@Batman2814 are you sure you are performing checks from devices in one network?

**Comment by Batman2814** [comment]
yes, I am. I'm doing it from devices that are connected to my Eero directly which is setup using Plain AdGuard Personal DNS where I'm using the browser on the Samsung TV I have.

**Comment by GulnazSalakhova** [comment]
Please confirm whether the issue still occurs with current results; otherwise, this issue can be considered stale.
